src/Inventory/Api/LineItemService.php

```diff
@@ -411,6 +420,17 @@ private function getRootAdUnit() {
 		return $adUnit;
 	}
 
+	private function setupCustomInventoryTargeting($adUnitName) {
+		if ($adUnitName === 'wka1b.iu') {
+			$adUnit = new AdUnitTargeting();
+			$adUnit->setAdUnitId(22279857691);
```

Comment:
It would be useful if we'd describe the magic numbers in this method for future us ;)